### DISCUSSION TITLE
Missing documentation on how to remove standalone certs

### DISCUSSION THREAD

**Discussion by JKHSDTV** [nginx-proxy/acme-companion#1136] [Q&A] [answered]
Bug template doesn't really apply, as this issue is about unclear/missing documentation in https://github.com/nginx-proxy/acme-companion/blob/main/docs/Standalone-certificates.md

The process to remove a cert added like that is not clear from the documentation. Presumably, there is no code to automatically detect if the config file has had lines removed when the reload through `signal_le_service` is triggered (unless there is, in which case it should be added to the documentation), so it would be good if the documentation would list what further steps would need to be taken to have certificates removed, and if you are supposed to manually delete them, whether further steps are needed to prevent unwanted renewal of certs no longer in use.

**Comment by buchdag** [comment, marked as answer]
Hi @JKHSDTV 

The service loop [detects if certificate (wether they're standalone or not) are no longer in active use and remove the corresponding symlinks from `/etc/nginx/certs`](https://github.com/nginx-proxy/acme-companion/blob/e9c01c99a5407ab197a1d8e0871d3e0c0b84a0dc/app/letsencrypt_service#L52-L123) on every execution (including manual trigger of `signal_le_service`).

Certificate in active use means a certificate existing in either the generated `/app/letsencrypt_service_data` file or the user created `/app/letsencrypt_user_data` file. Certificate not in active use are also no longer renewed.
